### DISCUSSION TITLE
Questions about seqr's Exomiser and Lirical Feature

### DISCUSSION THREAD

**Discussion by ghost** [broadinstitute/seqr#4053] [Q&A]
Hi seqr team,

I see that there is a feature for adding data from exomiser or lirical.  I'm not yet familiar with lirical, but we run exomiser on all of family groups we can.  Based on the original feature request:  https://github.com/broadinstitute/seqr/issues/2742 
And PR:  https://github.com/broadinstitute/seqr/pull/2989/files
I can get an idea of how the data should be formatted, but it's not super clear.  Could you provide any documentation for how we should be uploading the exomiser TSV files in seqr?

**Comment by hanars** [comment]
We are currently in the process of updating how we run these tools and what format seqr accepts, so I would not recommend adding this data at this time and would instead recommend you wait a couple months for the migration to be completed

**Reply by ghost** [reply]
Hi @hanars.  We're just looking for some documentation for how this is currently implemented as we have our own deployment.  I'm glad to see you all are working on this feature though.  We are gearing up to upgrade our pipeline to Exomiser v14 by next quarter.

Do you have any current documentation that you all provide for uploading the exomiser to seqr or even how you run exomiser?  Will there be any documentation in a few months once you upgrade/migrate to the new SOP for exomiser/lirical?

**Reply by hanars** [reply]
So no, we do not have current documentation, as what is currently live in seqr is beta functionality and not well supported. Once we feel confident that we are done iterating on the input format and have a stable pipeline, we will then add documentation. We hope this will be in a couple months but this is work being done in collaboration with another group so we do not necessarily have complete control of the timeline

**Reply by ghost** [reply]
Thanks @hanars!
